huntsman_dust/power_spectrum.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+import util_rad_profile
+from scipy import fftpack
+from scipy import optimize
+import matplotlib.pyplot as plt
+import huntsman_dust.util_plot as util_plot
+from huntsman_dust.detect_mask_sources import detect_mask_sources
+
+
+def power_spectrum(image):
+    image = image.astype(float)
+    psd2D = np.abs(fftpack.fftshift(fftpack.fft2(image)))**2
+    psd1D = util_rad_profile.azimuthalAverage(psd2D)
+    return psd1D
+
+
+if __name__ == '__main__':
+    image, img_masked, _, _ = detect_mask_sources(image_path='/Users'
+                                                  + '/amanchokshi/Desktop'
+                                                  + '/Huntsman/Data'
+                                                  + '/ngc6822_r.fits',
+                                                  slice_in=1500,
```

Comment:
Is it possible to remove these slice defaults? I'm afraid the masking only is working on a subset, but the power spectrum is computed over the entire image. Also, is it really necessary to provide RA, Dec and Obj_name? It seems like these are in the fits header. 

Basically only include paramaters that are relevant.

Reply:
I will remove the slice defaults soon. The obj_name is necessary to mask the galaxy. Ra and Dec only need to be supplied if there is no internet activity. They provide the centre about which the galaxy is masked.